.prettierrc.json

```diff
@@ -1,12 +1,11 @@
 {
-  "endOfLine": "auto",
+  "endOfLine": "lf",
   "useTabs": false,
-  "printWidth": 180,
+  "printWidth": 160,
   "tabWidth": 2,
   "singleQuote": true,
   "bracketSpacing": true,
   "jsxBracketSameLine": true,
   "trailingComma": "es5",
-  "semi": true,
-  "parser": "typescript"
```

Comment:
Why do we remove this parser?

Reply:
because prettier is the formatt for all file types.
in the context of markdown or json or yaml, what does typescript mean?
prettier infers the correct parser, there should be no need to set it manually

Reply:
got it